### ISSUE TITLE
[FEATURE] - Support virtual filesystems

### ISSUE THREAD

**Issue by TylerLeonhardt** [alefragnani/vscode-project-manager#500]
In preview (vscode-insiders only), there are now a concept of "Remote repositories" (see [release notes](https://github.com/microsoft/vscode-docs/blob/main/release-notes/v1_56.md#remote-repositories-remotehub) which allows you to instantly browse, search, edit, and commit to any GitHub repository directly from within VS Code without having to clone or have the repository locally.
RemoteHub is a new feature in vscode insiders... It allows you to open a virtual filesystem of a GitHub repo. 
1. Click the remote development button on the bottom right of vscode
2. Click RemoteHub: Open GitHub Repository...
3. Paste in a url of a GitHub repo and it's open a virtual filesystem of that repo
For example, my https://github.com/TylerLeonhardt/vscode-autountitledlanguage is represented as:
`vscode-vfs://github/TylerLeonhardt/vscode-autountitledlanguage`
I tried putting this in as the root path but no luck:
![image](https://user-images.githubusercontent.com/2644648/112577536-b62c0c80-8db1-11eb-838e-b8dac6898eb5.png)
my guess is that vscode-project-manager just doesn't handle this scheme properly and think's it's a file path.

**Comment by alefragnani** [comment]
Hi @TylerLeonhardt ,

Yep, this happens because its `scheme` is not `vscode-remote`, so it _thinks_ to be local. I think it would be fairly easy to update the extension to support it.

Just out of curiosity, is this new built-in Remote Hub extension based/the same one from @eamodio? I just noticed it has basically the same concept (and name) and its repo is now archived.

BTW, do you have any suggestion for its `icon`? 😬

**Comment by TylerLeonhardt** [comment]
Yep! @eamodio has it built in to VS Code now 😁

**Comment by TylerLeonhardt** [comment]
I think for an icon, it should be like...Something that represents a virtual filesystem like maybe the `cloud` or `globe` or `rss`

**Comment by alefragnani** [comment]
The `cloud` icon was the first one in my mind. I think it's a good choice.

Thank you

**Comment by eamodio** [comment]
Yeah, _Remote Repositories (RemoteHub)_ is based on my previous RemoteHub idea, but also expands on it greatly to encompass a much richer VS Code environment -- with editing and SCM support. And we're just getting started 😁

**Comment by alefragnani** [comment]
I knew it 😆

**Comment by alefragnani** [comment]
I'm wondering if I should use `cloud` or `remote` codicon for this 😬 

![Screen Shot 2021-05-07 at 13 19 28](https://user-images.githubusercontent.com/3781424/117479568-dca7a080-af36-11eb-88d5-4e13ac655a23.png)

BTW, I didn't understand why the extension was "internally" disabled for Virtual Workspaces. Just added the proper `capabilities` entry on `package.json` and it appears to be working just fine. I'm a bit confuse because it has the same `extensionKind` as my other Bookmarks extension, and that one wasn't disabled (and is working just fine 🎉 ). About that, I'm going to ask/comment on that tracking issue about _disabled functionalities_. If you could take a look/help, it would be great.

Well, proper support should be available in the next release. Stay tuned

**Comment by TylerLeonhardt** [comment]
> I'm wondering if I should use cloud or remote codicon for this 😬

Remote does look prettttty nice!

**Comment by alefragnani** [comment]
No problem about being disabled. It was just me being curious 😬. 

In the end is good that way because forces me (and other extension developer) to properly evaluate/test the extension, before enabling it.
